### PR TITLE
K8SPSMDB-256 - Update for release 1.5.0

### PR DESCRIPTION
**Pull request by tplavcic**
[K8SPSMDB-256](https://jira.percona.com/browse/K8SPSMDB-256)

### CODE REVIEW

File: deploy/bundle.yaml [hunk 1]
```diff
@@ -16,6 +16,9 @@ spec:
     - name: v1
       storage: true
       served: true
+    - name: v1-1-0
+      storage: false
+      served: true
```
Comment (cap1984) [inline deploy/bundle.yaml:21]: not sure whether we need this.
@delgod do we?

Comment (delgod) [reply deploy/bundle.yaml:21]: technically not
but let's add, reason - to have all versions consistent

**Review by delgod — APPROVED** [review]

File: deploy/bundle.yaml (hunk 1 above)
Comment (tplavcic) [reply deploy/bundle.yaml:21]: the point is I create bundle automatically, so if I'll need to add exception on every release it's a bit painful, so I vote to make it consistent and easy to maintain

**Review by hors — APPROVED** [review]

**Review by cap1984 — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| one-pod | passed |
| upgrade | passed |
| storage | passed |
| init-deploy | passed |
| monitoring | passed |
| self-healing | passed |
| limits | passed |
| upgrade-consistency | passed |
| monitoring-2-0 | failed |
| demand-backup | passed |
| scaling | passed |
| operator-self-healing | passed |
| security-context | passed |
| scheduled-backup | passed |
| arbiter | passed |
| service-per-pod | passed |
| liveness | passed |
| users | passed |

**Pull request merged**